packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/AdvancedCoinSettingsModal/AdvancedCoinSettingsModal.tsx

```diff
@@ -44,20 +46,25 @@ export const AdvancedCoinSettingsModal = ({ coin, onCancel }: AdvancedCoinSettin
         return null;
     }
 
+    const blockchain = useSelector(state => state.wallet.blockchain);
```

Comment:
Please don't forget to fix this hook conditional call 🙏

Reply:
🤦